test/helpers/TestFileLoader.ts

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs';
+import path from 'path';
+
+export namespace TestFileLoader {
+  export type FilterOption = { exclude?: string[]; include?: string[] };
+
+  const filter = (files: string[], options: FilterOption): string[] => {
+    return files.filter((file) => {
+      if (options.include) return isIncluded();
+      if (options.exclude) return !isExcluded();
```

Comment:
By the way, I would have written this including the namespace name. Like,

```ts
TestFileLoader.isIncluded();
```

Reply:
However, the code is currently a nested function and can only be called within the `filter` function, so it cannot attach namespace names as it is now.

If we want to attach the namespace name to the `isIncluded` function call, we would need to move the function outside of the nested function and add an export. This way, the function can be called externally, but I am concerned whether this might cause any issues.

Alternatively, is it possible to attach the namespace name without encountering the above problem?

Reply:
Oh, you're right.